### DISCUSSION TITLE
How to transition from postgress sql to mysql/mariadb ?

### DISCUSSION THREAD

**Discussion by ishanabdul3** [NarHakobyan/awesome-nest-boilerplate#87] [Q&A] [answered]
Can you please provide some steps or config change for using mysql/mariadb in typeorm

**Comment by NarHakobyan** [comment, marked as answer]
Hi @ishanabdul3, just edit configurations in ormconfig.ts and api-config.service.ts files. and recreate all migrations.

**Reply by NarHakobyan** [reply]
you must delete and recreate all migrations.

**Reply by NarHakobyan** [reply]
Use Typeorm's migration generator instead 😆
